packages/account-v2/src/containers/PersonalDetailsFormWithExample/PersonalDetailsFormWithExample.tsx

```diff
@@ -11,7 +11,8 @@ import { validateField } from '../../utils/validation';
 type TPersonalDetailsFormWithExampleValues = InferType<ReturnType<typeof getNameDOBValidationSchema>>;
 
 type TPersonalDetailsFormWithExampleProps = {
-    onConfirm: () => void;
+    error?: string;
```

Comment:
error is not used int he component

Reply:
Yes. In this PR it is not required. But the PROP is required for the component. I'll remove this from here